blockchain-source/src/cardano/block.rs

```diff
@@ -0,0 +1,51 @@
+use cardano_sdk::chain::{
+    AnyCbor, BlockAlonzo, BlockShelley, Header, HeaderVasil, MetadataSet, TransactionBodies,
+    TxIndexes,
+};
+use cbored::{CborRepr, DecodeError};
+
+#[derive(Clone, Debug, CborRepr, PartialEq, Eq)]
+#[cborrepr(structure = "array")]
+pub struct BlockVasil {
+    pub header: HeaderVasil,
+    pub tx_bodies: TransactionBodies,
+    pub tx_witnesses: AnyCbor,
```

Comment:
probably should add a comment explaining why this field is `AnyCbor`, and why this is needed.

Reply:
makes sense